examples/spring-web/src/main/resources/static/assets/script.js

```diff
@@ -0,0 +1,45 @@
+(async function() {
+  // Clear any state in the URL
+  history.pushState({}, "", "/");
+
+  // General API request handler
+  async function get(uri) {
+    return fetch(uri).then(res => {
```

Comment:
This `then` is what I would replace with an `await`, otherwise no point in the added `async`.